tests/e2e/lib.sh

```diff
@@ -78,6 +108,29 @@ deploy_central() {
     "$ROOT/${DEPLOY_DIR}/central.sh"
 }
 
+deploy_sensor_from_helm_charts() {
+    if [ -z $1 ]; then
+        die "deploy_sensor_from_helm_charts should receive a helm chart version\nusage: deploy_sensor_from_helm_charts <Chart version> <path to init bundle>"
+    fi
+	chart_version="$1"
+    if [ -z $2 ]; then
```

Comment:
```suggestion
    if [[ "$#" -ne 1 ]]; then
        die "deploy_sensor_from_helm_charts should receive a helm chart version\nusage: deploy_sensor_from_helm_charts <Chart version> <path to init bundle>"
    fi
	chart_version="$1"
    if [[ "$#" -ne 2 ]]; then
```
Does the same as what Gavin mentioned apply here? Unsure if that's the case but it may be better to have a look.

Reply:
Done